pytorch_lightning/utilities/xla_device.py

```diff
@@ -66,7 +66,8 @@ def _is_device_tpu() -> bool:
         Return:
             A boolean value indicating if TPU devices are available
         """
-
+        if os.getenv("XRT_SHARD_WORLD_SIZE") is not None:
```

Comment:
what are the values this could take on? is checking `if "XRT_SHARD_WORLD_SIZE"` in os.environ sufficient?

Reply:
Hey @ananthsub, Yes. PR is WIP/Draft, need to do some testing.